src/integrator_types.jl

```diff
@@ -5,7 +5,7 @@ mutable struct DEOptions{SType,CType}
     callback::CType
 end
 
-mutable struct ODEInterfaceIntegrator{uType,uPrevType,oType,SType,solType,algType} <: DiffEqBase.AbstractODEIntegrator
+mutable struct ODEInterfaceIntegrator{uType,uPrevType,oType,SType,solType,algType} <: DiffEqBase.AbstractODEIntegrator{Nothing, uType}
```

Comment:
Nothing is not the algorithm type.

Reply:
Then what is?

Reply:
Lol I messed up :P It was at the end, fixed now.